app/javascript/chat/__tests__/draw.test.jsx

```diff
@@ -0,0 +1,21 @@
+import { h } from 'preact';
+import { render } from '@testing-library/preact';
+import { axe } from 'jest-axe';
+import Draw from '../draw';
+
+describe('<Draw />', () => {
+  it('should have no a11y violations', async () => {
+    const { container } = render(<Draw />);
+    const results = await axe(container);
+
+    expect(results).toHaveNoViolations();
+  });
+
+  it('should render', () => {
+    const { queryByText } = render(<Draw />);
+
+    expect(queryByText('Connect Draw')).toBeDefined();
```

Comment:
These tests need to check that the elements found are not null. e.g.
```suggestion
    expect(queryByText('Connect Draw'))not.toBeNull();
```

Otherwise, the tests will pass all the time even if `queryByText` returns `null`.

![image](https://user-images.githubusercontent.com/833231/93110408-44953800-f683-11ea-8190-dfba0c6c3727.png)